core/domain/collection_services.py

```diff
@@ -73,7 +73,8 @@ def _migrate_collection_to_latest_schema(versioned_collection):
     Args:
         versioned_collection: A dict with two keys:
           - schema_version: str. The schema version for the collection.
-          - nodes: The list of collection nodes comprising the collection.
+          - nodes: list(CollectionNode). The list of collection nodes comprising
```

Comment:
Just to check, are these CollectionNode domain objects, or dicts representing collection nodes? If the latter, the type should be list(dict), and the docstring should be "The list of collection node dicts comprising the collection."